events/tests/test_organization_api.py

```diff
@@ -183,34 +189,47 @@ def assert_account_fields_exist(data):
     assert_fields_exist(data, fields)
 
 
+def assert_can_see_organization_users(data):
+    for field in organization_user_fields:
+        assert data.get(field) is not None
+
+
+def assert_cannot_see_organization_users(data):
+    for field in organization_user_fields:
+        assert data.get(field) is None
+
+
 @pytest.mark.django_db
-def test_admin_user_can_see_organization_users(organization, user, user_api_client):
-    organization.regular_users.add(user)
+def test_superuser_or_admin_can_see_organization_users(api_client, organization):
```

Comment:
Looks like this is testing only `super_user` not the organization admin. I suggest changing the name of the test to `test_super_user_can_see_organization_users`

Reply:
Still forgot a wrong name for this after splitting the test cases -> fixing

Reply:
Fixed